src/decisionengine_modules/NERSC/util/newt.py

```diff
@@ -82,24 +82,24 @@ def get_usage(self, username):
         self._login()
         iris_url = f"{self.newt_base_url}/account/iris"
         query = (
-            'accounts(username: \\"{}\\") {{ '
-            "   projectId, "
-            "   repoName, "
-            "   repoType, "
-            "   currentAlloc, "
-            "   usedAlloc, "
-            "   users {{ "
-            "       uid, "
-            "       name, "
-            "       firstname, "
-            "       lastname, "
-            "       middlename, "
-            "       userAlloc, "
-            "       userAllocPct, "
-            "       usedAlloc "
-            "   }} "
-            "}}"
-        ).format(username)
+            f'accounts(username: "{username}") {{ '
+            f"   projectId, "
+            f"   repoName, "
+            f"   repoType, "
+            f"   currentAlloc, "
+            f"   usedAlloc, "
+            f"   users {{ "
+            f"       uid, "
+            f"       name, "
+            f"       firstname, "
+            f"       lastname, "
+            f"       middlename, "
+            f"       userAlloc, "
+            f"       userAllocPct, "
+            f"       usedAlloc "
+            f"   }} "
+            f"}}"
```

Comment:
Since there are no variables here I think these could be simple literals ("..." instead of f"...")

Reply:
I agree. Not sure if you may have missed noticing, but I did mention in the PR description that I chose to add the `f` notation to keep it consistent. But as you pointed out, it makes total sense to skip using the `f` notation on these lines since they do not have any variables. This has been updated as suggested.